### PR TITLE
[MODORDERS-474] Unrelease encumbrances when order is reopened

### PR DESCRIPTION
**Pull request by damien-git**
### Purpose
[MODORDERS-474](https://issues.folio.org/browse/MODORDERS-474)
Added a new test to check encumbrances are released under certain conditions when an order is reopened.

### CODE REVIEW

**Comment by damien-git** [comment]
Screenshot_20210518_143726

File: acquisitions/src/main/resources/domain/mod-orders/features/check-encumbrances-after-order-is-reopened.feature [hunk 1]
```diff
@@ -0,0 +1,391 @@
...
+
+
+  Scenario: Close the order and release encumbrances
+    Given path 'orders/composite-orders', orderId
+    When method GET
+    Then status 200
+
+    * def orderResponse = $
+    * set orderResponse.workflowStatus = 'Closed'
+    * set orderResponse.compositePoLines[0].paymentStatus = 'Fully Paid'
```
Comment (AndreiMPV) [inline acquisitions/src/main/resources/domain/mod-orders/features/check-encumbrances-after-order-is-reopened.feature:152]: Please remove updating lines statuses. It must be updated programaticaly

**Comment by damien-git** [comment]
Screenshot_20210524_084928
Checked in testing.

**Comment by AndreiMPV** [comment]
Verified on testing
image

**Pull request merged**